moveit.rosinstall

```diff
@@ -1,34 +1,82 @@
 # This file is intended for users who want to build MoveIt! from source.
 # Used with wstool, users can download source of all packages of MoveIt!.
+
 - git:
     local-name: moveit
     uri: https://github.com/ros-planning/moveit.git
     version: master
+
 - git:
     local-name: moveit_msgs
-    uri: https://github.com/ros-planning/moveit_msgs.git
-    version: melodic-devel
+    # TODO(mlautman): update to ros-planning once the package has been ported
```

Comment:
```suggestion
    # TODO(mlautman): update to ros-planning once the package has been merged back in
```

and below

Reply:
seems like a nit

Reply:
seems like a nit

Reply:
we do nits all the time. you just gotta accept the commit suggestion... its more correct.